src/Resources/config/twig_hooks/shop/wishlist/index.yaml

```
@@ -0,0 +1,83 @@
+                priority: 0
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content':
+            header:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/header.html.twig"
+                priority: 100
+            form:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form.html.twig"
+                priority: 90
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form':
+            form:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections.html.twig"
+                priority: 0
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections':
+            items:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items.html.twig"
+                priority: 100
+            actions:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/actions.html.twig"
+                priority: 90
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections.actions':
+            collective_actions:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/actions/collectiveActions.html.twig"
+                priority: 100
+            global_actions:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/actions/globalActions.html.twig"
+                priority: 90
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections.items':
+            head:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head.html.twig"
+                priority: 100
+            body:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/body.html.twig"
+                priority: 90
+
+        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections.items.head':
+            select_all:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/selectAll.html.twig"
+                priority: 100
+            image:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/image.html.twig"
+                priority: 90
+            price:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/price.html.twig"
+                priority: 80
+            quantity:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/quantity.html.twig"
+                priority: 70
+            actions:
+                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/actions.html.twig"
+                priority: 60
```

in Sylius we have a convention to use step 100
```suggestion
        'bitbag_sylius_wishlist_plugin.wishlist.index.content.form.sections.items.head':
            select_all:
                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/selectAll.html.twig"
                priority: 400
            image:
                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/image.html.twig"
                priority: 300
            price:
                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/price.html.twig"
                priority: 200
            quantity:
                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/quantity.html.twig"
                priority: 100
            actions:
                template: "@BitBagSyliusWishlistPlugin/WishlistDetails/index/content/form/sections/items/head/actions.html.twig"
                priority: 0
```